lib/partition_setup.pm

```diff
@@ -212,6 +212,36 @@ sub addlv {
     assert_screen 'expert-partitioner';
 }
 
+sub addboot {
```

Comment:
I assume that this function is trying to add boot partion, right ?
In this case I would name a function `addbootpartion` or at least `addbootpart` to share same naming with addpart

Reply:
I was deciding among these names as you mentioned, but since we are dealing with ``partition_setup.pm`` lib it looks like the ``addboot`` is straightforward enough. Let's gather more feedback, thanks! :)

Reply:
make sense